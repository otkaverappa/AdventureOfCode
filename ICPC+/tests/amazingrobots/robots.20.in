20 20
####################
#X#...#...#...#....#
#.#.#.#.#.#.#.#.#.##
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#...#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#...#...#...#...#.##
#################.##
10
6 18 4 N
2 19 3 W
4 2 2 E
6 3 2 W
8 2 2 E
10 3 2 W
12 2 2 E
14 3 2 W
16 2 2 E
12 19 4 N
20 20
#.##################
#.#....#...#...#...#
#....#...#...#...#.#
######.###########.#
#..................#
#.###.##############
#..................#
##################.#
#..................#
#.###.##############
#..................#
#################..#
#..................#
#.#.##############.#
#..................#
##################.#
...................#
#.################.#
#.................X#
####################
10
3 4 2 E
17 1 2 E
18 19 2 S
17 19 2 S
14 19 2 S
5 6 3 S
11 6 3 N
11 18 3 S
13 4 3 S
4 7 2 N
