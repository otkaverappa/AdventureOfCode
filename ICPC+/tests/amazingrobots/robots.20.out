328
W
E
W
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
S
S
S
E
S
S
E
S
S
E
S
S
E
S
S
E
S
S
E
S
S
S
S
E
N
E
N
E
N
W
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
E
E
E
E
N
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
E
E
E
E
E
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
N
E
E
E
E
N
E
E
S
E
S
E
E
E
E
E
E
E
E
E
E
N
N
N
W
S
S
S
W
W
S
E
S
W
S
W
N
W
W
S
S
E
S
W
S
W
N
W
W
S
S
E
S
W
S
W
N
W
W
W
S
W
W
S
S
N
N
N
